packages/cloud_firestore/README.md

```diff
@@ -60,6 +60,19 @@ class BookList extends StatelessWidget {
 }
 ```
 
+Running a transaction
```

Comment:
I would put a colon here for consistency with the other examples in this section